### PR TITLE
Editorial nits and suggestions

### PR DESCRIPTION
**Pull request by glyn**
This PR adopts some editorial nits and suggestions from draft PR https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/275.

### CODE REVIEW

File: draft-ietf-jsonpath-base.md [hunk 1]
```diff
@@ -697,17 +692,17 @@ in the table below:
...
-Note that processing the name selector potentially requires matching strings against
-strings, with those strings coming from the JSONPath and from member
-names and string values in the JSON to which it is being applied.
+Note that processing the name selector requires comparing the converted member name string
+with member name strings in the JSON to which the selector is being applied.
 Two strings MUST be considered equal if and only if they are identical
 sequences of Unicode scalar values. In other words, normalization operations
-MUST NOT be applied to either the string from the JSONPath or from the JSON
-prior to comparison.
+MUST NOT be applied to either the converted member name string from the JSONPath or to
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:704]: Not sure I understand what "converted" is here.  Member name string parsed from the JSONPath?  Or is there some other conversion going on?

File: draft-ietf-jsonpath-base.md [hunk 2]
```diff
@@ -1313,6 +1310,9 @@ appear in the list.
 Note that any node matched by more than one selector is kept
 as many times in the nodelist.
 
+Where a selector can produce a nodelist in more than one possible order, the
+selector may produce a nodelist in a distinct order each time it appears in the child segment.
```
Comment (cabo) [inline draft-ietf-jsonpath-base.md:1314]: Is this just about appearance?  Or evaluation?

File: draft-ietf-jsonpath-base.md (hunk 1 above)
Comment (glyn) [reply draft-ietf-jsonpath-base.md]: Fair enough. I was trying to refer to the result of the conversion described just above in this section. I've fixed this in 55f19c2f13149210c10b19bcb1909f9864570708.

File: draft-ietf-jsonpath-base.md (hunk 2 above)
Comment (glyn) [reply draft-ietf-jsonpath-base.md]: I thought evaluation was clear, but perhaps not. Clarified in de0b902dca07cd68106d73b19b84b6c45c2cf2c6.

**Comment by glyn** [comment]
@cabo This PR picked up some commits from another PR. I'm going to close it and open another PR.

**Comment by glyn** [comment]
@cabo FYI I fixed the two commit SHAs referenced by this PR to point at the corresponding commits in PR #279.

**Pull request closed without merging**